src/components/BlockCombinator/index.js

```diff
@@ -54,7 +63,45 @@ function BlockCombinator({
     setLogicBlocks([]);
   }, [mapId]);
 
+  useEffect(() => {
+    try {
+      if (executingBlock) {
+        const indexes = executingBlock.split("-");
+
+        if (executingBlock !== "end") {
+          paintCurrentBlock(indexes, "#ffa500", "5px");
+        }
+        if (previousBlock.current.length !== 0) {
+          paintCurrentBlock(previousBlock.current, "black", "2px");
+        }
+
+        executingBlock === "end"
+          ? (previousBlock.current = [])
+          : (previousBlock.current = indexes);
+      }
+    } catch (error) {
+      navigate("/error", { state: { error: error } });
```

Comment:
```suggestion
      navigate("/error", { state: { error } });
```
위와 같이 줄여서 표현할 수 있을것 같아요.